### DISCUSSION TITLE
微服务、容器与容器云-从入门到放弃

### DISCUSSION THREAD

**Discussion by ycrxun** [ycrxun/ycrxun.github.io#45] [Announcements]
https://ronggle.com/2017/microservice-container-and-container-cloud/ 

 Search Engine Oriented Programming.

**Comment by wolfdu** [comment]
吃瓜测试组

**Comment by ycrxun** [comment]
@wolfdu 吃瓜测试组

**Comment by MingJunDuan** [comment]
踩个issues
